### PR TITLE
Lt 1805 application major editing

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: applications/tests/tests_application_status.py [hunk 1]
```diff
@@ -14,87 +15,108 @@
...
-        permission_denied_user = UserOrganisationRelationship.objects.get(organisation=other_organisation).user
-        permission_denied_user_headers = {
-            "HTTP_EXPORTER_USER_TOKEN": user_to_token(permission_denied_user),
-            "HTTP_ORGANISATION_ID": other_organisation.id,
-        }
+    @parameterized.expand(
+        [
+            status
+            for status, value in CaseStatusEnum.choices
+            if status != CaseStatusEnum.APPLICANT_EDITING
```
Comment (andytaylorkainos) [inline applications/tests/tests_application_status.py]: can you turn it into a `status not in [...]` - makes it a little easier on the eyes

File: applications/tests/tests_application_status.py [hunk 2]
```diff
@@ -14,87 +15,108 @@
...
+               and status != CaseStatusEnum.WITHDRAWN
+        ]
+    )
+    def test_exporter_set_application_status_failure(self, new_status):
+        """ Test failure in setting application status to any status other than 'Applicant Editing' and 'Withdrawn'
+        as an exporter user.
 
-        data = {"status": "something_stupid"}
-        response = self.client.put(self.url, data=data, **permission_denied_user_headers)
+        """
```
Comment (andytaylorkainos) [inline applications/tests/tests_application_status.py]: remove blank line above pls

File: applications/tests/tests_application_status.py (hunk 1 above)
Comment (stephanieboyd) [reply applications/tests/tests_application_status.py]: Done

File: applications/tests/tests_application_status.py (hunk 2 above)
Comment (stephanieboyd) [reply applications/tests/tests_application_status.py]: Removed

**Pull request merged**